eth2/beacon/tools/fixtures/loading.py

```
@@ -0,0 +1,111 @@
+import os
+from pathlib import Path
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    Tuple,
+)
+
+from eth_utils import (
+    to_tuple,
+)
+from ruamel.yaml import (
+    YAML,
+    YAMLError,
+)
+
+from eth2.beacon.helpers import (
+    compute_epoch_of_slot,
+)
+from eth2.configs import (
+    Eth2Config,
+)
+
+from eth2.beacon.tools.fixtures.test_file import (
+    TestFile,
+)
+
+
+#
+# Eth2Config
+#
+def generate_config_by_dict(dict_config: Dict[str, Any]) -> Eth2Config:
+    for key in list(dict_config):
+        if 'DOMAIN_' in key:
+            # DOMAIN is defined in SignatureDomain
+            dict_config.pop(key, None)
+
+    dict_config['GENESIS_EPOCH'] = compute_epoch_of_slot(
+        dict_config['GENESIS_SLOT'],
+        dict_config['SLOTS_PER_EPOCH'],
+    )
+    return Eth2Config(**dict_config)
+
+
+def get_config(root_project_dir: Path, config_name: str) -> Eth2Config:
+    # TODO: change the path after the constants presets are copied to submodule
+    path = root_project_dir / 'tests/eth2/fixtures'
+    yaml = YAML()
+    file_name = config_name + '.yaml'
+    file_to_open = path / file_name
+    with open(file_to_open, 'U') as f:
+        new_text = f.read()
+        try:
+            data = yaml.load(new_text)
+        except YAMLError as exc:
+            print(exc)
+            raise
```

can we just `raise exc` here, rather than printing it?

i like to not force code to `print` when i can avoid it